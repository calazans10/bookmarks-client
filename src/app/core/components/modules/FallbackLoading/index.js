import React from 'react';
import './index.scss';

export const FallbackLoading = () => (
  <div className="fallback-loading">
    <p>Loading...</p>
  </div>
);

export default FallbackLoading;
