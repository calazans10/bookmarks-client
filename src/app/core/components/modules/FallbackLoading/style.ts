import styled from 'styled-components';

export const Container = styled.div`
  display: flex;
  position: fixed;
  z-index: 999;
  top: 0;
  left: 0;
  align-items: center;
  justify-content: center;
  width: 100%;
  height: 100%;
  background: #fff;
  color: #333;
`;
