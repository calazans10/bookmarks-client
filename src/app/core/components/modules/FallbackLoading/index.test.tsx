import React from 'react';
import { render } from '@testing-library/react';
import FallbackLoading from '.';

describe('FallbackLoading', () => {
  it('renders without crashing', () => {
    const { container } = render(<FallbackLoading />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
