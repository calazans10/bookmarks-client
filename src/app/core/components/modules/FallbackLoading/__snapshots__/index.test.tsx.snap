// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FallbackLoading renders without crashing 1`] = `
<div
  class="sc-AxjAm foMttB"
>
  <p>
    Loading...
  </p>
</div>
`;
